nodeinstaller/internal/constants/configuration-qemu-tdx.toml

```diff
@@ -18,10 +18,6 @@ cpu_features="-vmx-rdseed-exit,pmu=off"
 default_vcpus = 1
 default_maxvcpus = 0
 default_bridges = 1
-# On TDX, when lowering this, the patch:
-# packages/by-name/qemu-tdx-static/0004-hw-x86-load-initrd-to-static-address.patch
-# needs to be updated accordingly.
-default_memory = 2048
```

Comment:
@msanft: Changing this does not seem to require the anticipated change in the qemu patch. Do you know why that's the case?

Reply:
Changing this as in removing it?

I think the [default value](https://github.com/kata-containers/kata-containers/blob/eb1cf792de5b62a6d1897602018bb67db965850d/src/runtime/pkg/katautils/config-settings.go.in#L64) is 2048, which would explain this.

Reply:
Take a look at `platforms.go`, I'm changing the default to 512.

Reply:
I'm not exactly sure. I would expect it not to work. Did you check what memory argument is being passed to QEMU?

Reply:
Yes: the expected sum of 512 and the pod limit.

Reply:
Which should - in almost all cases - amount to something < 2Gi, right?

Reply:
Yes. Should I check whether verification fails for large VMs?

Reply:
It should only fail for small VMs, really, as we try to load the initrd to an address larger than the maximum VM memory. Unless I miss some QEMU sanity check anywhere, this should break.

Reply:
Well, at least the openssl test does not fail on TDX with this change.

Reply:
I see, and that's what confuses me here. 
This will need some thorough investigation, but I'm not yet sure when I have the capacity for it.